test/e2e/Makefile

```diff
@@ -38,7 +38,8 @@ help:  ## Display this help
 ## --------------------------------------
 ## Binaries
 ## --------------------------------------
-
+DEFAULT_VELERO_IMAGE := velero/velero:main
```

Comment:
Why we need this?

Reply:
It's not needed since I've changed code to pass version from main Makefile to E2E Makefile, so don't have to parse from this default value.